istio-control/istio-autoinject/files/injection-template.yaml

```diff
@@ -115,10 +117,12 @@ template: |
     - --datadogAgentAddress
     - "{{ .ProxyConfig.GetTracing.GetDatadog.GetAddress }}"
   {{- end }}
-    - --proxyLogLevel
-    - "{{ annotation .ObjectMeta `sidecar.istio.io/debug` .Values.global.proxy.logLevel}}"
-    - --log_output_level
-    - '{{ .Values.debug }}'
+  {{- if .Values.global.proxy.logLevel }}
```

Comment:
This removes the ability to override the log level per pod - which is quite useful.

Reply:
Can we make this `sidecar.istio.io/logLevel` then instead of `debug`?

Reply:
sure. 

As a general rule - we want the annotations to match the keys used in values.yaml, and eventually to generalize so anything in values.yaml (== mesh config) can be patched by local annotations.